mappings/net/minecraft/client/gui/widget/EntryListWidget.mapping

```diff
@@ -5,6 +5,8 @@ CLASS cwm net/minecraft/client/gui/widget/EntryListWidget
 			ARG 3 mouseY
 		METHOD render (IIIIIIIZF)V
 			ARG 1 index
+			ARG 2 y
+			ARG 3 x
```

Comment:
Is this the right order? Usually it's x, y.

Reply:
Yes blame mojang

Reply:
Okay, then.